### PR TITLE
Added reference to guidelines notice in dragout menu section

### PR DESCRIPTION
**Pull request by NonameSLdev**
Dragout menus on web are forbidden by the material guideline rules, so it's good to let people know and have them choose by themselves. I added a simple card for that.

### CODE REVIEW

**Comment by tomscholz** [comment]
The pages are compile via jade. Please edit the template here https://github.com/Dogfalo/materialize/blob/master/jade/mobile/mobile_content.html

File: mobile.html [hunk 1]
```diff
@@ -124,6 +124,9 @@ <h2 class="header">Navbar</h2>
         </p>
         <h4>Drag Out Menu</h4>
         <p>This plugin includes several options for customizing the menu. See <a href="side-nav.html#options">Plugin Options</a> for details.</p>
+        <div class="card-panel amber lighen-1">
```
Comment (tomscholz) [inline mobile.html]: Its´s `lighten` not `lighen`

**Comment by tomscholz** [comment]
And I guess you should also add the warning to the sidenav page.

**Comment by NonameSLdev** [comment]
@tomscholz Sure thing, made another pullrequest. Close this one?

**Comment by tomscholz** [comment]
@NonameSLdev I guess you can close this one and make a new PR 😃

**Comment by tomscholz** [comment]
@NonameSLdev 
- Add left class to icon
- Remove second span and replace with b tag
- lighten not lighen
:)

**Comment by NonameSLdev** [comment]
@tomscholz Silly me :P I was in a hurry so I excuse myself. Fixing on both pages and making a new PR.

**Pull request closed without merging**